### DISCUSSION TITLE
send Bin file via EspNow

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#837] [Q&A] [answered]
How to send Bin file via espnow and your API

For bin file data order is important 
all data is important  . No missing data will  accepted .
No repeat data will accepted 

so i  am confuse  can we achieve this ?

**Comment by pschatzmann** [comment, marked as answer]
I think you will need to give a little bit more background: for me your question does not make any sense!
All audio data: encoded or not is always binary. If you check the my default settings, the use_send_ack is activated by default, so you should never get any lost or repeated data!
